### ISSUE TITLE
EntryToken token card not showing up

### ISSUE THREAD

**Issue by colourful-land** [AlphaWallet/alpha-wallet-android#1600]
1. Enable TS debugging.

2. Drop this file to the AlphaWallet directory on the device;
https://github.com/AlphaWallet/TokenScript/blob/access-cards-from-web/tests/EntryToken.xml

3. Observe in "TokenScript Management" the newly dropped in file works (it's the last one)

![Screenshot_20200914-185512_AlphaWallet](https://user-images.githubusercontent.com/548435/93065369-fa7c7880-f6bb-11ea-80d9-ed2c59fe01eb.jpg)

4. Using the watch wallet address `0xC067A53c91258ba513059919E03B81CF93f57Ac7`
 and Rinkeby is selected. There is no Entry Token in the wallet.

**Comment by tomekalphawallet** [comment]
I think the QR code should not be there. 

The top right corner should be a place where we have a question mark. It can trigger a modal window. And it does not necessarily have to be there. If a token issuer don't have any content, it can be hidden. 

<img width="1989" alt="Screenshot 2020-09-16 at 10 16 24" src="https://user-images.githubusercontent.com/51817359/93312697-cf06a480-f807-11ea-955e-6359aee49a73.png">

**Comment by tomekalphawallet** [comment]
You can combine Signature Details and Smart Contract Address in the same modal
<img width="476" alt="Screenshot 2020-09-16 at 10 45 03" src="https://user-images.githubusercontent.com/51817359/93314141-aed7e500-f809-11ea-81ca-4edf041e95f7.png">

**Comment by JamesSmartCell** [comment]
FYI I think the contract diamond is only displayed on debug builds. It's an incredibly useful debugging tool.

If it's appearing in release build I'll remove it so it only appears on debug build. Are you seeing this on the play store version?

**Comment by colourful-land** [comment]
> If it's appearing in release build I'll remove it so it only appears on debug build. Are you seeing this on the play store version?

Yes it is in the playstore version, but even a user should have a way to find out the current contract information (like address) so that he can do things like comparing and finding out if the token he has is a fake one (happens so frequently these days) by comparing that with the official information.

**Comment by colourful-land** [comment]
Hi @ChintanRathod :

Don't let all the previous discussion distract you. What you need to do is the following:

-----

1. Make sure there is a question mark icon once the user clicks a token in the Wallet Tab and opens it. There are 2 scenarios to deal with: the situation that the user clicks a ERC20 token like DAI, where the user sees the Token Card; the situation that the user clicks a NFT token like ENS, where the user sees a list of tokens. In both scenarios, the icon should be in the same location.

2. Make sure old "diamond icon" is replaced with the new icon.

3. When the user clicks that icon, shows a table-based information set instead of a QR code like this:

<img width="476" alt="Screenshot 2020-09-16 at 10 45 03" src="https://user-images.githubusercontent.com/51817359/93314141-aed7e500-f809-11ea-81ca-4edf041e95f7.png">

(Of course, the "Signature Infromation" is only there if the token has a TokenScript. Be aware, that if the TokenScript is unsigned - i.e. one being debugged - the "Signature Information" should 𝑠𝑡𝑖𝑙𝑙 be there, except with a message that the TokenScript isn't signed).

**Comment by tomekalphawallet** [comment]
![Zrzut ekranu 2020-09-22 o 14 13 44](https://user-images.githubusercontent.com/51817359/93880496-dd5c3100-fcdd-11ea-9a2d-f909cd455475.png)

So we have a question mark and the lock icons - depending on the context described by Weiwu. 

https://zpl.io/2GLGPEY
https://zpl.io/a3L1rv1
